packages/amplify-provider-awscloudformation/src/root-stack-builder/stackSynthesizer.ts

```diff
@@ -0,0 +1,45 @@
+import { ISynthesisSession, Stack, LegacyStackSynthesizer } from '@aws-cdk/core';
```

Comment:
Should this file be in core? I'm assuming every category would need this?

Reply:
also change to stack-synthesizer.ts

Reply:
it depends since storage isnt using it and API also

Reply:
But auth is using it correct? We should standardize if we should use _toCloudFormation or this method.

Reply:
Discussed offline that we'd be using synthesizer and consolidating it in one place as a part of the auth PR (which is coming up next).